definitions/variable/industry/tag_iron_commodities.yaml

```diff
@@ -0,0 +1,15 @@
+- Iron Commodity:
```

Comment:
Is it planned that there will be additional iron commodities? If no, better to avoid this tag-list.

Reply:
Further refined/manufactured steel commodities could be added to that list in the future I suppose. The worldsteel association reports production of ingots, semis, tubes, plates etc. in their [annual yearbooks](https://worldsteel.org/wp-content/uploads/Steel-Statistical-Yearbook-2018.pdf). But the tag-list can be created once there is more than one product.